syntax = "proto3";
message Request{
    float value =1;
}
message Response{
    float result=1;
    string name = 2;
    string on_off_status=3;
    repeated string object_comands =4;
}
service ActuatorGRPC{
    rpc TunrnOn(Request)    returns (Response){};
    rpc TunrnOff(Request)   returns (Response){};
    rpc ModStatus(Request)  returns (Response){};
    rpc SeeStatus(Request)  returns (Response){};
    rpc SeeComands(Request) returns (Response){};
}